python/pyspark/ml/tests/test_algorithms.py

```diff
@@ -333,7 +333,7 @@ def test_linear_regression_with_huber_loss(self):
     from pyspark.ml.tests.test_algorithms import *  # noqa: F401
 
     try:
-        import xmlrunner
+        import xmlrunner  # type: ignore[import]
```

Comment:
I'm unsure if we also should include the `tests/` folder

Reply:
I think we should. Next to examples, these are the biggest chunks of code that show (most of the time) intended usage. This is a great sanity check.